segmenters/nlp/NLTKSentencizer/__init__.py

```diff
@@ -0,0 +1,103 @@
+from typing import Dict, List
+
+from jina.executors.segmenters import BaseSegmenter
+
+
+class NLTKSentencizer(BaseSegmenter):
+    """
+    Segment text into sentences using :class:`nltk.PunktSentenceTokenizer`
+    for the specified language.
+
+    Example:
+
+    >>> sentencizer = NLTKSentencizer() # doctest:+ELLIPSIS
+    NLTKSentencizer@...[I]:post_init may take some time...
+    NLTKSentencizer@...[I]:NLTK english sentence tokenizer ready.
+    NLTKSentencizer@...[I]:post_init may take some time takes ... seconds (...s)
+    >>> text = "Today is a good day. Can't wait for tomorrow!"
+    >>> sentencizer.segment(text)
+    [{'text': 'Today is a good day.', 'offset': 0, 'location': [0, 20]}, {'text': "Can't wait for tomorrow!", 'offset': 1, 'location': [21, 45]}]
+
+
+    :param language: default='english'. Lowercased language name to initialize the sentence tokenizer, accepted languages are listed in :attr:`SUPPORTED_LANGUAGES`.
+    :type language: str
+    :param args: Additional positional arguments
+    :param kwargs: Additional keyword arguments
+    """
+
+    SUPPORTED_LANGUAGES = [
+        'czech',
+        'danish',
+        'dutch',
+        'english',
+        'estonian',
+        'finnish',
+        'french',
+        'german',
+        'greek',
+        'italian',
+        'norwegian',
+        'polish',
+        'portuguese',
+        'russian',
+        'slovene',
+        'spanish',
+        'swedish',
+        'turkish',
+    ]
+
+    def __init__(self, language: str = 'english', *args, **kwargs):
+        """Set constructor"""
+        super().__init__(*args, **kwargs)
+        self.language = language
+        self.sent_tokenizer = None
```

Comment:
Avoid setting `sent_tokenizer`. This is something very Jina specific to avoid serializing expensive models i